### DISCUSSION TITLE
Using ReEDS for VPP analysis?

### DISCUSSION THREAD

**Discussion by hudson-wagner** [NREL/ReEDS-2.0#162] [General]
Hi all, I'm a master's student preparing to write my thesis in 2024. I'm looking to write the thesis about virtual power plants (VPPs) and their effects on the larger power grid and transmission / distribution systems. Would ReEDS be an appropriate tool for modeling VPPs at larger scales? I am just beginning to familiarize myself with the model.

**Comment by vcelloho** [comment]
You could model VPPs in ReEDS by making appropriate changes to the model. We currently don't have a VPP technology as part of the model or a way for the model to see contributing technologies as a combined unit and there would need to be work to ensure that the demand side and supply sides of a VPP are reflected.

The question of appropriateness of ReEDS will depend on the spatial granularity that you want to focus on. ReEDS by default is modeled at 134 regions for the US and doesn't represent the distribution system. Additionally is your analysis focused on investments in VPPs (capacity expansion question) or comparing the operations of supply and demand resources combined vs not combined into a VPP?

**Comment by xueboyu-123** [comment]
hi,I also plan to write a paper about virtual power plants (VPPs) ,can i chat with you

**Reply by hudson-wagner** [reply]
Hi, I’ve moved my focus away from ReEds for my thesis. But if you’re still interested in talking feel free to reach out.
